src/Paket.Core/Installation/RestoreProcess.fs

```diff
@@ -324,18 +326,31 @@ let createPaketPropsFile (lockFile:LockFile) (cliTools:ResolvedPackage seq) (pac
                  yield "    </ItemGroup>"])
             |> fun xs -> String.Join(Environment.NewLine,xs)
 
+    let disableImplicitFSharpCore =
+        if packages |> Seq.exists (fun ((_,name),_,_) -> name = FSharpCore) then
+            """
+            <!-- Disable automagic references for F# dotnet sdk -->
+            <!-- This will not do anything for other project types -->
+            <!-- see https://github.com/fsharp/fslang-design/blob/master/tooling/FST-1002-fsharp-in-dotnet-sdk.md -->
+            <DisableImplicitFSharpCoreReference>true</DisableImplicitFSharpCoreReference>
+            <DisableImplicitSystemValueTupleReference>true</DisableImplicitSystemValueTupleReference>
+            """
+        else
+            ""
+
     // When updating the PaketPropsVersion be sure to update the Paket.Restore.targets which checks this value
     let content =
         sprintf """<?xml version="1.0" encoding="utf-8" standalone="no"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
     <PropertyGroup>
         <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
         <PaketPropsVersion>5.185.3</PaketPropsVersion>
-        <PaketPropsLoaded>true</PaketPropsLoaded>
+        <PaketPropsLoaded>true</PaketPropsLoaded>%s
```

Comment:
What's the %s at the end here?

Reply:
it's a big sprintf ".... %s ....".   I also the parameter coule of lines below

Reply:
Aah, guess context got lost using a cellphone to see the code...